### PR TITLE
Practice with Streams

### PR DESCRIPTION
**Pull request by AlexandruIliescu**
- created 2 classes 
- solved some exercises

### CODE REVIEW

File: src/session17/Product.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package session17;
+
+public class Product {
+
+    private String name;
+    private double price;
+
+    public Product(String name, double price) {
```
Comment (rhacp) [inline src/session17/Product.java:8]: `Test`
Please consider to remove this constructor.

Comment (AndreiMarius95) [inline src/session17/Product.java:8]: "TEST"
I checked you, it's correct

File: src/session17/StreamsPractice.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+//        printEvenNumbersWithStream(numbers);
+//        printStringsByLengthBiggerThan(strings, 5);
+//        printStringsLength(strings);
+//        printProductPrices(products);
+//        printProductIncreasedPrices(products);
+//        printFlatMap(listOfLists);
+//        printSumOfNumbers(numbers);
+//        printSortedProductsByPrice(products);
+//        printProcessedNumbers(numbers);
+        printAnyMatchFruit(strings);
```
Comment (AlexandruIliescu) [inline src/session17/StreamsPractice.java:32]: Please consider to remove commented code.

File: src/session17/Product.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package session17;
+
```
Comment (Daniel7Ionescu) [inline src/session17/Product.java:2]: hello world

File: src/session17/StreamsPractice.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public class StreamsPractice {
+
+    public static void main(String[] args) {
+        List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+        List<String> strings = Arrays.asList("apple", "banana", "peach", "watermelon");
```
Comment (GabiPascu) [inline src/session17/StreamsPractice.java:12]: Consider having Bob in the strings List

File: src/session17/Product.java (hunk 3 above)
Comment (AlexandruIliescu) [reply src/session17/Product.java:2]: hello back

File: src/session17/StreamsPractice.java (hunk 4 above)
Comment (SamuelParvu) [inline src/session17/StreamsPractice.java:12]: no devil fruit 😿

**Review by SamuelParvu — APPROVED** [review]
great changes

**Comment by GabiPascu** [comment]
Amazing, please proceed

**Pull request merged**